ui/appui-react/src/appui-react/statusfields/SelectionCount.scss

```diff
@@ -6,8 +6,8 @@
   gap: var(--iui-size-2xs);
   display: flex;
   align-items: center;
-  min-width: var(--iui-size-2xl);
-  padding-left: var(--iui-size-2xs);
+  min-width: 3.75rem;
```

Comment:
Could use the [`ch` unit](https://caniuse.com/ch-unit).  If this is accounting for the icon as well you could try something like `calc(var(--iui-icon-size) + 4ch)` for example.

Reply:
I think this would be a viable option but it seems that the `--iui-icon-size` (or `--iui-size-icon`) doesn't exist. Therefore using `ch` units does not really make sense when we have an icon. I will leave rem (as it is the unit used by iTwinUI) for now. We can change it later.

Reply:
Sorry, I didn't look up the actual variable and just gave that as an example before I signed off for the day.